src/AbstractSQLCompiler.ts

```diff
@@ -146,13 +147,15 @@ export type CoalesceNode = [
 	...UnknownTypeNodes[],
 ];
 export type ToJSONNode = ['ToJSON', AnyTypeNodes];
+export type AnyNode = ['Any', UnknownTypeNodes];
```

Comment:
If you can split the addition of the `Any` node out to a separate PR that'd be very helpful to keep PRs as small as possible and much easier/quicker to review, meaning all the PRs can get merged sooner and with less issues around rebase conflicts

Reply:
Will do. Commit history is a mess rn